build_tools/configure_stm32f4.sh

```diff
@@ -49,6 +49,16 @@ case $2 in
       export PATH_TO_LINKER_SCRIPT="`realpath ../build_tools/stm32f407-libopencm3.ld`"
     fi
     ;;
+  
+  stm32f446)
+    echo "Building for STM32F446"
+    if [ "$1" == "cmsis" ]; then
+      export PATH_TO_LINKER_SCRIPT="`realpath ../build_tools/stm32f446-cmsis.ld`"
+    else
+      echo "Unsupported lib. Only 'cmsis' is supported for board 'STM32F446'"
+      exit 1
```

Comment:
We can think about dropping this change as #80 does a similar check on the CMake side.